### PR TITLE
179376124 copy page and nav

### PR DESCRIPTION
**Pull request by knowuh**
First half of  [179376124 - Page Navigation and copy and move page](https://www.pivotaltracker.com/story/show/179376124)

In order to keep the PRs managable.

Next up will be:
- LARA controller action
- Adding the navigation element to the page stories.

### CODE REVIEW

**Comment by knowuh** [comment]
I am going to put this in for review now. 

It is the first-half of the work for [179376124  - Page Navigation and copy and move page](https://www.pivotaltracker.com/story/show/179376124)

File: lara-typescript/src/section-authoring/containers/api-container.tsx [hunk 1]
```diff
@@ -26,7 +26,8 @@ export const APIContainer: React.FC<IAPIContainerProps> = (props) => {
     <UserInterfaceProvider>
       <APIContext.Provider value={APIProvider}>
         <QueryClientProvider client={queryClient}>
-          {/* Handy for debugging ReactQuery: <ReactQueryDevtools /> */}
+          Handy for debugging ReactQuery:
+          <ReactQueryDevtools />
```
Comment (emcelroy) [inline lara-typescript/src/section-authoring/containers/api-container.tsx:30]: Should this be commented out again?

**Review by emcelroy — APPROVED** [review]
This looks good, thanks! I noted a few minor possible issues in inline comments. One other little thing I'm wondering is if `lara-typescript/src/section-authoring/components/page-nav-menu/page-nav-menu.tsx` should be `lara-typescript/src/section-authoring/components/page-nav-menu/index.tsx`? I see that pattern used elsewhere in the lara-typescript folder.

File: lara-typescript/src/section-authoring/api/api-types.ts [hunk 2]
```diff
@@ -178,6 +179,7 @@ export type APIPageItemDeleteF = (args: {pageId: PageId, pageItemId: ItemId}) =>
 export type APIPageItemUpdateF = (args: {pageId: PageId, sectionItem: ISectionItem}) => Promise<ISectionItem>;
 export type APIPageItemCopyF = (args: {pageId: PageId, sectionItemId: ItemId}) => Promise<ISectionItem>;
 
+
```
[the remark this answers is not in the document]
Comment (knowuh) [reply lara-typescript/src/section-authoring/api/api-types.ts:182]: Oh, good catch, I am not sure why it didn't complain...

[the remark this answers is not in the document]
Comment (knowuh) [reply lara-typescript/src/section-authoring/api/api-types.ts:182]: Oh -- Yeah, forgot to run `npm run build` -- so I didn't catch the linting error.

**Comment by knowuh** [comment]
I am going to leave `page-nav-menu/page-nav-menu.tsx` for now although I have also seen `index.tsx` used in many cases.

**Review by emcelroy — APPROVED** [review]
Looks good 👍

**Pull request merged**